canonical/prometheus-k8s-operator
# Fix #386 - alert rules disappear if k8s cluster gets rebooted

## Issue


This PR fixes #386 


## Testing Instructions


- Deploy Prometheus
- Deploy a charm that implements `MetricsEndpointProvider` interface (Zinc)
- Relate both charms
- Check that the alert rules provided by Zinc are in Prometheus
- restart microk8s
- Check that the alert rules provided by Zinc are in Prometheus ;-)
